ATSAMD21/luos_hal.c

```diff
@@ -76,18 +76,6 @@ void LuosHAL_Init(void)
 
     //Com Initialization
     LuosHAL_ComInit(DEFAULTBAUDRATE);
-
```

Comment:
This commit should be vanish by removing commit related to the add of those debug pins.